styles.css

```diff
@@ -209,6 +532,61 @@ nav ul {
   flex: 1;
 }
 
+.popup_body p {
+  order: 4;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 30px;
+  color: #344563;
+}
+
+.close_mobile {
+  display: flex;
```

Comment:
[OPTIONAL] I noticed this property declared multiple times in this project, please consider creating a class for it to make it truly reusable and to avoid unnecessary code duplication which might cause performance issues.

e.g 
```

.flex {
  display: flex;
}
```

You can put this class in your HTML tags where necessary.

Reply:
Understood, thank you very much for the suggestion